From a7846260bb1a595e5316b971d13e85f0f28039c7 Mon Sep 17 00:00:00 2001
From: Yi Yao <yi.yao@intel.com>
Date: Fri, 21 Dec 2018 16:40:18 +0800
Subject: [PATCH 0906/1214] x86: add missing Konfig option for turbo decompress

There losts an kconfig opiton for the previous turbo decompress patch

Change-Id: I6a788d1e8f356f683c345b78c4da51c6f5717da0
Tracked-On: PKT-1621
Signed-off-by: Yi Yao <yi.yao@intel.com>
---
 arch/x86/Kconfig | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/x86/Kconfig b/arch/x86/Kconfig
index 1f2f90b..5026f54 100644
--- a/arch/x86/Kconfig
+++ b/arch/x86/Kconfig
@@ -208,6 +208,13 @@ config X86
 	select VIRT_TO_BUS
 	select X86_FEATURE_NAMES		if PROC_FS
 
+config TURBO_BEFORE_DECOMPRESS
+        bool "Turbo CPU freq before kernel decompress"
+        depends on X86_64
+        ---help---
+          Make CPU frequency start to run at P0 state before
+          kenrel decompress of system boot.
+
 config INSTRUCTION_DECODER
 	def_bool y
 	depends on KPROBES || PERF_EVENTS || UPROBES
-- 
2.7.4

